src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/DES/DESCipherTests.cs

```diff
@@ -0,0 +1,190 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Text;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Encryption.Des.Tests
+{
+    public static class DesCipherTests
+    {
+        // These are the expected output of many decryptions. Changing these values requires re-generating test input.
```

Comment:
Yes the current tests use RFC2268 for RC2+ECB and FIPS81 for DES (ECB and CBC) in addition to comparing against values from .net desktop. I could not find public test vectors for RC2+CBC.